CommonKit/Sources/CommonKit/Models/FileResult.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  FileResult.swift
+//  
+//
+//  Created by Stanislav Jelezoglo on 06.03.2024.
+//
+
+import UIKit
+
+public enum FileType {
+    case image
+    case video
+    case other
+    
+    public var isMedia: Bool {
+        self == FileType.image || self == FileType.video
```

Comment:
Use switch. If we add a new case in the future we will not forget to apply it here. Yes, this property is written right after the cases list, but who knows, maybe someone will move it to an extension